.gitattributes

```diff
@@ -0,0 +1,3 @@
+/.* export-ignore
+/tests export-ignore
+/phpunit.xml.dist export-ignore
```

Comment:
what is this?

Reply:
It excludes folders which will be put in dist archive and therefore will be downloaded and extracted to vendor directory by composer. These files are not required for applications which uses this bundle, they are only necessary for developing the bundle itself. See https://git-scm.com/docs/gitattributes#_export_ignore

Reply:
yea, that sounds like its going to be confusing when you download from github and these files are missing

Reply:
It is a best practice for bundles/libraries, because tests directory could contain php files with unsafe access to the filesystem etc, there was a known phpunit vulnerability several years ago, since that time many libraries exclude these files from the distribution, see examples in very popular libraries:
* https://github.com/sebastianbergmann/phpunit/blob/master/.gitattributes
* https://github.com/doctrine/orm/blob/2.10.x/.gitattributes